helm-charts/konk-service/values.yaml

```diff
@@ -51,6 +51,7 @@ ingress:
   #      - chart-example.local
 
 konk:
+  create: true
```

Comment:
Changing this default will create a bunch of konks on existing services that are currently sharing one.